specification/common-types/resource-management/v5/types.json

```diff
@@ -371,7 +450,7 @@
         "id": {
           "type": "string",
           "format": "arm-id",
-          "description": "Fully qualified resource ID for the resource. E.g. \"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}\"",
+          "description": "Fully qualified resource ID for the resource. Ex - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}",
```

Comment:
>Ex [](http://example.com/codeflow?start=72&length=2)

Was e.g. no good here, but ok elsewhere? Why 'Ex'? And not 'Example'? (in the spirit of keeping docs simple/obvious and avoiding excessive unclear abbreviations)